tests/ffi/disk/test.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -euvx
+#!/bin/bash -evx
```

Comment:
I would prefer the code below if you don't mind.

Reply:
which code below ?
https://github.com/containers/qm/pull/553#discussion_r1756694384
Do you suggest this ? 

#!/bin/bash

Reply:
Yes but I suggest convert a single line of code to something more mature.